### ISSUE TITLE
Please add a Changelog

### ISSUE THREAD

**Issue by natterstefan** [davodesign84/react-native-mixpanel#142]
Hi @davodesign84,

first of all, thanks for the module. I appreciate your work and effort. One question though, can you add a changelog or release tags or anything that can help to see the progress, breaking changes and other notes about new versions? That would be very helpful when one decides to upgrade your module or not. For instance I had 0.0.16 installed and now the latest version is 1.0.2. I then ask myself what the breaking change was (0.x->1.x) that caused the 1.x release, or if there is any.

Thank you very much.

**Comment by wkoutre** [comment]
@natterstefan While I agree, keep in mind standard versioning:

Major.minor.patch

Major: likely has breaking changes
Minor: likely new features, backwards compatible
Patch: small additions and bug fixes

**Comment by natterstefan** [comment]
@wkoutre yeah of course and I agree. The breaking change note was more or less an assumption I made in my comment.

**Comment by wkoutre** [comment]
Gotcha, gotcha. Agreed 100% though -- no matter how small, always nice to have a `changelog`!

**Comment by marssantoso** [comment]
They're actually written in the commit message. Like this last one ([commit 23455eb](https://github.com/davodesign84/react-native-mixpanel/commit/23455eb5661f620d86cca7f1b27c21dcd9d94441)).

Took me sometime to find it too.

**Comment by danielmartinprieto** [comment]
We were going to open the same issue and we saw this one, as it came out today with my team. It'd be great to have a changelog along with enforcing semantic versioning!
